### ISSUE TITLE
report this

### ISSUE THREAD

**Issue by King-Fox-Yt** [luckydog7/Funkin-android#608]
this is stolen and not giving lucky dog credit
https://play.google.com/store/apps/details?id=xredfox.friday.funkin
![Screenshot_20210502-233524](https://user-images.githubusercontent.com/83482176/116829974-2e4ed480-ab9f-11eb-8db4-a100e9e13053.png)

**Comment by DabKnight** [comment]
We already know about it.

**Comment by WyxosYT** [comment]
Yeah I came across it while searching for FNF rippoffs for a vid lol

**Comment by ROBIKALIVE13** [comment]
> Yeah I came across it while searching for FNF rippoffs for a vid lol

lul

**Comment by Justine101213k** [comment]
Lol Chill Bro Xd

**Comment by Justine101213k** [comment]
I reported
